charts/vault-monitoring/ci/default-values.yaml

```diff
@@ -0,0 +1,87 @@
+replicaCount: 1
+
+vault:
+  serverName: vault.example.com
+  port: 443
+  portName: https
+  ca_path: /etc/vault/ssl/ca.crt
+  ip: 10.1.2.3
+  scheme: https
+  ca: ""
+  auth:
+    mount_path: auth/kuberntees
+    role: metrics
+  service:
+    type: externalName
+    selector: {}
+  serviceMonitor:
+    labels:
+      k8s.example.com/prometheus: kube-prometheus
+
+prometheusRules:
+  enabled: true
+  rules:
```

Comment:
It could make sense to move a part of these to values.yaml since some of them (ie. `VaultSealed`) should be created on an install w/o values.

Also, this file in `ci/` should really only contain the minima needed to pass ci.

Reply:
Yeah its in the wrong file.

Reply:
Also it would generate a lot of content in the readme tho

Reply:
> Also it would generate a lot of content in the readme tho

It's enough if the readme just has a hint at the existence of the rules and maybe a link to the file.